dev/MRTCore/azure-pipelines.yml

```diff
@@ -302,6 +264,15 @@ jobs:
       artifactName: 'signedbinaries_x64'
       downloadPath: '$(Build.ArtifactStagingDirectory)\raw'
 
+  - task: DownloadBuildArtifacts@0
```

Comment:
I don't suppose there's a good way to parameterize these downloads/copies per architecture like you did above?

Reply:
We can still do those in parallel. However, those are very simple tasks of copying files into final places. Doing them on separate jobs and sync/publish/download just adds extra complexity and overhead.